docs/integrations/business-intelligence/connecting-to-metabase.md

```diff
@@ -35,26 +33,26 @@ You are now ready to run Metabase with your Firebolt databases.
 
     ![Metabase home page](../../assets/images/Metabase_home.png)
 
-2. From the **Settings** menu, select the **Admin** link:
+2. From the **Settings** menu, click the **Admin** link:

```

Comment:
We don't use "click", we use "select" for accessibility reasons and consistency throughout the doc set.

Reply:
Oh okay, I'll change it back. To me, "select" only makes sense if you're picking one out of many options, such as from a list, or if you're highlighting some text. But "Select the 'Save' button" sounded weird to me. If you're driving a car, you wouldn't "select" the 'On' button.

But whatevs, not ready to die on this hill (especially if it's consistent everywhere else).

Reply:
Another thought - the word "link" is used in this doc, which is technically true because it is a web page, but feels wrong in the age of web apps. To me, I reserve "link" for inline text (such as Google search results, or prose) and would prefer to say something else for UI elements. Like "Go to the Admin page." But just a nit.

Reply:
Yeah, "link" shouldn't be there. Should just be "From x menu, select z." We can fix these up as time allows